bin/plugin/commands/changelog.js

```diff
@@ -836,19 +833,32 @@ function getUniqueByUsername( items ) {
 }
 
 /**
- * Produces the formatted markdown for the full time contributors section of
- * the changelog output.
+ * Excludes users who should not be included in the changelog.
+ * Typically this is "bot" users.
+ *
+ * @param {IssuesListForRepoResponseItem[]} pullRequests List of pull requests.
+ * @return {IssuesListForRepoResponseItem[]} The list of filtered pull requests.
+ */
+function skipUsers( pullRequests ) {
```

Comment:
Nit: Maybe `skipCreatedByBots` for now? It was a bit unclear what it does in other places.

Reply:
The `skipUsers` function was inspired by #38538. Probably a rename would help clarify its purpose, such as `skipCreatedByBots` or `skipFakeContributors`.

Reply:
Yeh actually for now renaming to include the "bots" term would be helpful. I was originally thinking it might be a catch all solution but there isn't a use case for that at the moment so let's keep things simple.

Reply:
✅